src/controllers/user.js

```diff
@@ -90,17 +90,7 @@ const getSingleUser = async (req, res) => {
 // eslint-disable-next-line consistent-return
 const updateUser = async (req, res) => {
   const bodyParams = Object.keys(req.body); // ['name', 'email', 'password']
-  const allowedParams = [
-    'firstName',
-    'lastName',
-    'username',
-    'phoneNumber',
-    'age',
-    'gender',
-    'nationality',
-    'refugee',
-    'profilePhoto',
-  ];
+  const allowedParams = constants.VALID_USER_KEYS;
   const emptyOrWhiteSpace = /^\s*$/;
   try {
     const User = await UserModel.findById(req.user.userId);
```

Comment:
`User` should be camelCased in ALL files; I'm adding this upon Halit's previous request.